aviatrix/resource_aviatrix_gateway.go

```diff
@@ -385,6 +385,30 @@ func resourceAviatrixGateway() *schema.Resource {
 				DiffSuppressFunc: DiffSuppressFuncString,
 				Description:      "A list of monitored instance ids separated by comma when 'monitor gateway subnets' feature is enabled.",
 			},
+			"idle_timeout": {
```

Comment:
Why do we need both a boolean field and value field? Isn't the value field alone enough?
I see two situations:
1)  User supplies a value, assume boolean is true
2) User does not supply value, assume boolean is false